folding-schemes/examples/multiple_inputs.rs

```diff
@@ -0,0 +1,156 @@
+#![allow(non_snake_case)]
```

Comment:
Right, in fact is not even _'add'_ because it does additions and multiplications, I've renamed it to `multi_inputs.rs` and the circuit is now `MultiInputsFCircuit`.